### DISCUSSION TITLE
Git Bash tearing issue

### DISCUSSION THREAD

**Discussion by eltsu7** [charmbracelet/bubbletea#793] [Q&A] [answered]
Hey, I'm using wish server that runs a game I made. I've noticed that when I connect to it with git bash the screen is flashing and tearing on every update. Has anyone else seen this and found an answer?

More information: 
- Windows commandline and powershell work fine. 
- Git bash is using xterm, but also cygwin was experiencing the same flickering. 
- Happens inside and outside windows terminal, so I don't think that is the issue.

This is a video of git bash and powershell connected to the program:

https://github.com/charmbracelet/bubbletea/assets/23433034/0792842e-0682-4746-9186-357361e137ba

**Comment by eltsu7** [comment, marked as answer]
Answered in here: https://github.com/git-for-windows/git/discussions/4541

Tldr; they use different ssh clients. Using OpenSSH client from git bash doesn't cause problems.
